Engine/ac/gamestate.cpp

```diff
@@ -647,7 +648,8 @@ void GameState::WriteForSavegame(Common::Stream *out) const
     }
     out->WriteInt32( text_min_display_time_ms);
     out->WriteInt32( ignore_user_input_after_text_timeout_ms);
-    out->WriteInt32( ignore_user_input_until_time);
+    auto ignore_user_input_until_time_ms = std::chrono::duration_cast<std::chrono::milliseconds>(ignore_user_input_until_time - AGS_Clock::now());
+    out->WriteInt32( ignore_user_input_until_time_ms.count() );
```

Comment:
I'd wish we did not save this at all, does not make much sense to restore this value after loading a game IMO. It's my oversight that I did not remove it from new save format (there were too many variables to investigate so I got lazy and used same function again)

Reply:
That sounds reasonable. On the other hand, this actually works now, if someone manages to save in the short time input is ignored, it will continue to be ignored. :)